core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedElementNameUtil.java

```diff
@@ -90,7 +74,8 @@ static String getName(Object element) {
                         "cannot obtain the name of the parameter or field automatically. " +
                         "Please make sure you compiled your code with '-parameters' option. " +
                         "If not, you need to specify parameter and header names with @" +
-                        Param.class.getSimpleName() + " and @" + Header.class.getSimpleName() + '.');
+                        Param.class.getSimpleName() + " and @" + Header.class.getSimpleName() +
+                        " and @" + Attribute.class.getSimpleName() + '.');
```

Comment:
How about rephrasing a little bit?

Before:

```
"If not ...... " ... + '.'
```

After:

```
"Alternatively, you could specify the name explicitly in the annotation."
```

Reply:
I agree. 
I replace it!